src/middleware.ts

```diff
@@ -1,7 +1,12 @@
-import { authMiddleware } from "@clerk/nextjs";
+import { clerkMiddleware } from "@clerk/nextjs/server";
+import { NextResponse } from "next/server";
 
-export default authMiddleware({
-  publicRoutes: ["/"],
+export default clerkMiddleware(async (auth, req) => {
+  if (req.nextUrl.pathname === "/dashboard" && !auth().userId) {
+    auth().protect();
+
+    return NextResponse.next();
+  }
```

Comment:
Lets encourage the use of the pattern matcher helper. 
```suggestion
const isProtectedRoute = createRouteMatcher(['dashboard/(.*)']);

export default clerkMiddleware(async (auth, req) => {
  if (isProtectedRoute(req) && !auth().userId) {
    auth().protect();

    return NextResponse.next();
  }
```

Reply:
Great catch, oddly I believe in other places I was doing just that so idk why i did this lol